scripts/ci/_utils.sh

```diff
@@ -37,7 +37,7 @@ function verbose_docker {
     docker "${@}"
 }
 
-function initialize_breeze_environment {
+function initialize_common_environment {
```

Comment:
Would you mind adding a comment on what is the "common environment"? It will help others to understand the purpose ;)

Reply:
Sure :). Done